draft-ietf-jose-hpke-encrypt.md

```diff
@@ -243,6 +245,7 @@ When encrypting, the inputs to HPKE Seal operation are set as follows:
 - kdf_id: Depends on the JOSE-HPKE algorithm used.
 - aead_id: Depends on the JOSE-HPKE algorithm used.
 - info: By default, an empty string. Application MAY specify some other value.
+TBD: The existing JWE specifications do not provide a mechanism to include application context as AAD. Addressing this limitation exclusively for HPKE, while not applying similar measures to other algorithms, introduces an asymmetry in how messages are bound to their origin. This inconsistency could create potential vulnerabilities by differing security assurances across cryptographic algorithms. Why is this required just for JWE HPKE ?
```

Comment:
Issue #13 seems related to this.

Reply:
Yes, it makes sense to write a separate draft, and I can start drafting it. It would be great if you and Oliver could co-author it.